### ISSUE TITLE
Several outstanding issues with new deals and staging engine

### ISSUE THREAD

**Issue by 10d9e** [application-research/estuary#798]
_From Ryan at Portrait_

**Describe the bug**

- I tried uploading a [1GB test file](https://testfiledownload.com/), shows up as 1MiB in Estuary. (image)
- I believe there’s one staging zone (1630) for our account (User ID:69)
  - Can one have multiple staging zones now?
    - We had more than one before, now there’s only one.
  - The staging zone has 775 records, we have 2444 pins:
    - Is the difference (minus 1 working deal) the data that is lost?
  - Our only staging zone is 110.60 MiB, which is way less than expected.
    - Files pages says 22.90 GiB
      - Think this doesn’t take the data loss into account?
  - I uploaded a 54mb video, ID: 068795608, through the UI, which has not moved to the staging zone. Waited +- 15mins.
  
  
![image](https://user-images.githubusercontent.com/1556714/208094787-8dd9dfe1-754d-4c2a-a6ec-015a065b888b.png)

**Comment by en0ma** [comment]
> Can one have multiple staging zones now?
> * We had more than one before, now there’s only one.

Yes, we are currently running a staging zone reconstructive to fix and persist staging zones so they never go missing again

>The staging zone has 775 records, we have 2444 pins:
>Is the difference (minus 1 working deal) the data that is lost?

The aim of the reconstruction is to order the contents in the way there were pinned into their zones. When it is done, we believe your records should match your pins

>Our only staging zone is 110.60 MiB, which is way less than expected.
Files pages says 22.90 GiB
Think this doesn’t take the data loss into account?

The reconstruction started yesterday, it will address these issues

>I uploaded a 54mb video, ID: 068795608, through the UI, which has not moved to the staging zone. Waited +- 15mins.

That is because of the reconstruction, it will eventually catch up and your contents will show up in their respective zones
